### DISCUSSION TITLE
Focus 3 / XR Elite Passthrough Modes

### DISCUSSION THREAD

**Discussion by korejan** [korejan/ALVR#74] [Announcements]
The latest nightly builds of [ALXR](https://github.com/korejan/ALXR-nightly/releases/latest) now has support for passthrough modes for Focus 3 & XR Elite. Controls are Left Menu + A (blend mode) or B (mask mode)

**Comment by maxxfoos** [comment]
Hello there, great work so far ! . which version should i install on the quest pro of the apk and which for the server pc ?
thank you so much
